dockstore-webservice/src/main/java/io/dockstore/webservice/core/Container.java

```diff
@@ -459,11 +459,12 @@ public void setToolname(String toolname) {
         this.toolname = toolname;
     }
 
-    @JsonProperty("tool_path")
+    @JsonProperty
```

Comment:
Originally it was to avoid confusion, since setting it to be "tool_path" makes it seem like the field actually means something, when really it doesn't, since toolpath is generated on the fly from the path and toolname.

Though putting more thought into it, it is probably best to keep it as tool_path to maintain consistency among other json values, so I will change it back.